test/performance/odp_l2fwd.c

```diff
@@ -1411,8 +1408,8 @@ static void init_port_lookup_tbl(void)
 		int pktio_idx     = odp_pktio_index(pktio);
 		int dst_port      = find_dest_port(rx_idx);
 
-		if (pktio_idx < 0 || pktio_idx >= MAX_PKTIO_INDEXES) {
-			ODPH_ERR("Bad pktio index %i\n", pktio_idx);
+		if (pktio_idx < 0) {
+			ODPH_ERR("Reading pktio index failed: %i\n", pktio_idx);
```

Comment:
Could be useful in debugging to print something about the failing handle, e.g. `odp_pktio_to_u64()`.

Reply:
Added device name to error print in V2.